atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/compact/BackgroundCompactor.java

```diff
@@ -60,10 +61,12 @@
             return Optional.empty();
         }
 
+        CompactPriorityCalculator compactPriorityCalculator = new CompactPriorityCalculator(transactionManager);
```

Comment:
Would rather use a factory method in BackgroundCompactor so we don't have to create this and then pass it into the constructor - it's an implementation detail.